src/main/java/fish/payara/extras/diagnostics/util/Obfuscation.java

```diff
@@ -166,13 +174,39 @@ private void obfuscateAttribute(Element element, String attributeName) {
                 if (hostsReplacements.containsKey(keywordAttribute)) {
                     obfuscatedReplacement = hostsReplacements.get(keywordAttribute);
                 } else {
-                    obfuscatedReplacement = OBFUSCATED_CHANGE + obfuscatedCounter;
+                    obfuscatedReplacement = OBFUSCATED_CHANGE + (hostsReplacements.size() + 1);
                     hostsReplacements.put(keywordAttribute, obfuscatedReplacement);
-                    obfuscatedCounter++;
                 }
                 element.setAttribute(attributeName, obfuscatedReplacement);
             }
         }
     }
 
+    public static void obfuscateLogData(Path sourceFile, Path destinationFile) throws IOException {
```

Comment:
Is there a reason to be static?

Reply:
If the function was not static, line 165-167 in `LogCollector.java` would not work

`if (obfuscateEnabled){`
             `Obfuscation.obfuscateLogData(file, resolvedDestination);`
 #`}else {`